### ISSUE TITLE
Object Oriented Programming

### ISSUE THREAD

**Issue by sodhancha** [subashshakya1/JEWEL_SOFT#2]
Please use principles of Object Oriented Programming.
Rewrite the code in this project to follow Object Oriented best practices:

Resource:
http://code.tutsplus.com/tutorials/object-oriented-php-for-beginners--net-12762

**Comment by subashshakya1** [comment]
thank you for the information... I'll do my best.. :)

**Comment by sodhancha** [comment]
Have a look at this class https://gist.github.com/jonashansen229/4534794
Try implementing it in this repository and commit.

**Comment by subashshakya1** [comment]
yes i have gone through that repository. I'l be implementing it soon..
